sotodlib/mapmaking/demod_mapmaker.py

```diff
@@ -385,6 +379,206 @@ def write(self, prefix, tag, m):
 
         return oname
 
+def setup_demod_map(noise_model, shape=None, wcs=None, nside=None,
+                    comm=mpi.COMM_WORLD, comps='TQU', split_labels=None,
+                    singlestream=False, dtype_tod=np.float32,
+                    dtype_map=np.float32, recenter=None, verbose=0):
+    """
+        Setup the classes for demod mapmaking and return
+        a DemodMapmmaker object
+    """
```

Comment:
Please format your docstrings a bit more consistently with the rest of sotodlib. The extra indentation is not our practice, should be:
```
    """
    Setup the ...
```

Same for make_demod_map, below.